include/pros/rtos.hpp

```diff
@@ -22,8 +22,10 @@
 #ifndef _PROS_RTOS_HPP_
 #define _PROS_RTOS_HPP_
 
+#include "rtos/task.h"
```

Comment:
This cannot be included in `pros/rtos.hpp`, since the rtos headers aren't added to the kernel template, so this will result in an include file not found error.

Reply:
Addressed in https://github.com/purduesigbots/pros/pull/677/commits/26f77c86587f6ac3e6fb0991f71622199ac848f1.